pkg/controller/pod_volume_backup_exec.go

```diff
@@ -0,0 +1,76 @@
+package controller
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"sync"
+	"time"
+
+	arkv1api "github.com/heptio/ark/pkg/apis/ark/v1"
+)
+
+type Progress struct {
+	PercentDone float64 `json:"percent_done"`
+}
+
+func (c *podVolumeBackupController) RunBackupCommand(cmd *exec.Cmd, req *arkv1api.PodVolumeBackup) (string, string, error) {
+
+	var stdoutBuf, stderrBuf, stdoutStatBuf bytes.Buffer
+	stdoutIn, _ := cmd.StdoutPipe()
+	stderrIn, _ := cmd.StderrPipe()
+
+	var errStdout error
+	stdout := io.MultiWriter(os.Stdout, &stdoutBuf, &stdoutStatBuf)
+	stderr := io.MultiWriter(os.Stderr, &stderrBuf)
+
+	err := cmd.Start()
+	var wg sync.WaitGroup
+
+	wg.Add(1)
+	go func() {
+		_, errStdout = io.Copy(stdout, stdoutIn)
+		wg.Done()
```

Comment:
Might want to consider changing this to:
   defer wg.Done() 

and move that up to beginning of the function the go routine is executing.